sommelierd

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S ruby --disable-all
+#!/usr/bin/env ruby
```

Comment:
`#!/usr/local/bin/ruby`?

Reply:
I don't think this is a showstopper either...